cmd/tools/genrpcwrappers/main.go

```diff
@@ -426,18 +403,19 @@ func writeTemplatedMethod(w io.Writer, service service, impl string, m reflect.M
 		}
 	}
 
-	panicIfErr(template.Must(template.New("code").Parse(text)).Execute(w, fields))
+	codegen.FatalIfErr(codegen.GenerateTemplateToWriter(tmpl, fields, w))
 }
 
-func writeTemplatedMethods(w io.Writer, service service, impl string, text string) {
+func writeTemplatedMethods(w io.Writer, service service, impl string, tmpl string) {
 	sType := service.clientType.Elem()
 	for n := 0; n < sType.NumMethod(); n++ {
-		writeTemplatedMethod(w, service, impl, sType.Method(n), text)
+		writeTemplatedMethod(w, service, impl, sType.Method(n), tmpl)
 	}
 }
 
-func generateFrontendOrAdminClient(w io.Writer, service service) {
-	writeTemplatedCode(w, service, `
+func generateFrontendOrAdminClient(w io.Writer, service service) error {
+	writeTemplatedCode(w, service, `// Code generated by cmd/tools/genrpcwrappers. DO NOT EDIT.
```

Comment:
All templates here also can be extracted to separate files, but it is too much work. Maybe later.